viewing_party/party.py

```diff
@@ -1,23 +1,163 @@
 # ------------- WAVE 1 --------------------
+from statistics import mean, mode
 
 def create_movie(title, genre, rating):
-    pass
 
-# -----------------------------------------
-# ------------- WAVE 2 --------------------
-# -----------------------------------------
+    if title and genre and rating:
+    
+        movie = {"title": title,
+    "genre": genre,
+    "rating": rating,
+    }
+
+        return movie

```

Comment:
Amazing job completing your first project Maggie! This is a matter of personal preference and what reads easiest for you, but since the variable movie is only used in the return statement, we could factor it out by directly returning the new dictionary. Something like this:
```
return {
        "title": title,
        "genre": genre,
        "rating": rating,
}

```